nmoscommon/webapi.py

```diff
@@ -561,7 +607,7 @@ def __init__(self, oauth_config=None):
         if _config.get('fix_proxy') == 'enabled':
             self.app.wsgi_app = ProxyFix(self.app.wsgi_app)
 
-    def add_routes(self, routesObject, basepath):
+    def add_routes(self, routesClass, basepath):
```

Comment:
Is this really a class? The access of __class__ below makes it look like the original naming was correct, but I haven't looked explicitly at what is passed into it.